### DISCUSSION TITLE
New version release, v11.13.0

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#128] [Archived]
New major update
MaterialFX has been reviewed in every single package/class/resource to give the user the best experience possible
Enjoy!

P.S: demo still not available in GitHub releases since I'm having some issues with Jlink at the moment

**Comment by mfdewit** [comment]
Great work! I just got familiar with this project recently and I like it so far. Integration with Scenebuilder is also great.
The new virtualizedfx implementation also solved a scrollbar issue that was present on v11.12.

I'm looking forward on using it in my projects. Thanks and keep up the good work!

**Comment by mahfoud09** [comment]
MFXDatePicker and MFXFilterCombo not found at scene builder and when i set the textFormatter on MFXTextField this exception Appear
Formatter is already used in other control
